frontend/src/pages/pipelines/global/experiments/useDefaultExperiment.ts

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import { ExperimentKFv2, PipelinesFilterOp } from '~/concepts/pipelines/kfTypes';
+import useFetchState, { FetchState, FetchStateCallbackPromise } from '~/utilities/useFetchState';
+
+/**
+ * Fetch the first created experiment and check it's name to make sure it's default experiment
+ */
+const useDefaultExperiment = (): FetchState<ExperimentKFv2 | null> => {
+  const { api } = usePipelinesAPI();
+
+  const getDefaultExperiment = React.useCallback<
+    FetchStateCallbackPromise<ExperimentKFv2 | null>
+  >(async () => {
+    const response = await api.listExperiments(
+      {},
+      {
+        filter: {
+          predicates: [
+            // eslint-disable-next-line camelcase
+            { key: 'name', operation: PipelinesFilterOp.EQUALS, string_value: 'Default' },
```

Comment:
should this be sorted by creation date. i am sure it is by default, but to be safe

Reply:
We had a discussion here https://redhat-internal.slack.com/archives/C06SMDJGZC2/p1721400438442669 and decided to go with this filter instead of assuming the creation date.

Reply:
cool cool